### DISCUSSION TITLE
During the live session is it possible to go back to the desired place using videojs?

### DISCUSSION THREAD

**Discussion by brightlightkim** [videojs/video.js#7784] [Q&A]
I'm using the live data stream now.. I'm not sure how to use the videojs to go back.

Right now, when it's live the progress bar is even not clickable. 

I want to implement a service like a Youtube Livestream that it can go back.. 

Do video.js has the internal technology with it??

When I searched on the Google, I found 0 answer related this.

**Comment by gkatsev** [comment]
It depends a bit on the stream. Not all live streams allow seeking. You also need to set [`liveui: true` as an option for video.js](https://videojs.com/guides/options/#liveui)

**Reply by brightlightkim** [reply]
How can I use the seekbar for the live stream on videojs? I'm using the EdgeCast API and it allows seeking I think since it allows a pause and keep recording the live one at the back end.

**Reply by brightlightkim** [reply]
Thank you for your fast reply btw. :)

**Reply by kuolemaaa** [reply]
@gkatsev can you elaborate on that? I'm trying different HLS live streaming sources and some of them allows me to seek back and return to live time ( `live` label is places on the right of the player and it turn red when it follow the live with a seek bar in the middle of controlbar) and with some of them i dont have any seek bar in the controlbar with the `live` label on the left of the player without any (red)led/indicator.
I would like to know how to generate a HLS stream that allows seekback and return to live.

Honestly, i really dont understand why and how it is a feature of the source stream and not from the player since I guess that a client can seekback in the time interval that he already watched (namely the buffer that the client already downloaded and was shown on screen):

this is videojs 8.23.3


with seekbar:
![seekbar_present](https://github.com/user-attachments/assets/d43b37d1-8c9f-4128-a5d4-7b42caf321c5)
without seek bar:
![without_bar](https://github.com/user-attachments/assets/5938530b-c953-4995-b4f2-320596e9bb4b)


is there something i can check in the HLS in order to differentiate them?
